### PR TITLE
Fail tests when there are prop type errors

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA: no QA, tests should pass.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7721/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7722/

File: karma.conf.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ module.exports = function(config) {
       'jujugui/static/gui/build/app/assets/javascripts/d3-min.js',
 
       'jujugui/static/gui/src/app/components/**/*.js',
+      'jujugui/static/gui/src/test/required-props.js',
```
Comment (hatched) [inline karma.conf.js]: I think a comment here explaining that this needs to go ahead of any tests would be helpful in the future

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7735/

**Comment by huwshimi** [comment]
Thanks @frankban and @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**